jupyterhub/bases/jupyterhub/configs/jupyterhub_config.py

```diff
@@ -1,11 +1,28 @@
 from kubernetes.client import V1Capabilities, V1SELinuxOptions
 spawner = c.OpenShiftSpawner
-def mcs_selinux_profile(spawner, pod):
+
+def idh_apply_pod_profile(spawner, pod):
+  """
+  This method is responsible for modifying the final notebook pod spec before submitting it to the k8s api
+  Modifying the pod based on the V1Pod and V1PodSpec avaialable in the kubernetes python API
+  https://github.com/kubernetes-client/python/blob/master/kubernetes/docs
+  """
+  # Pod container zero is the 'notebook' container with the env var we need to check
+  nb_container_env = pod.spec.containers[0].env
+
   # Apply profile from singleuser-profiles
   apply_pod_profile(spawner, pod)
   if spawner.gpu_mode and spawner.gpu_mode == "selinux" and \
        spawner.extra_resource_limits and "nvidia.com/gpu" in spawner.extra_resource_limits:
     # Currently a bug in RHEL Docker 1.13 whereby /dev IPC dirs get inconsistent MCS
     pod.spec.security_context.se_linux_options = V1SELinuxOptions(type='nvidia_container_t',level='s0')
+
+  for item in nb_container_env:
+    if item.name == "enable_host_ipc":
+      # This serviceAccount should have the same permission as the 'default' serviceAccount
+      #   with an applied SCC for hostIPC access
+      pod.spec.service_account = 'jupyterhub-nb-hostipc-access'
```

Comment:
This wasn't required when we tested using the okd environment so you'll want to confirm that it is required for PSI

Reply:
I have commented this block out for now, we can remove it in a subsequent PR if it works in prod.